src/twisted/python/_release.py

```diff
@@ -39,14 +40,66 @@
 
 
 def runCommand(args, **kwargs):
-    """Execute a vector of arguments.
+    """
+    Execute a vector of arguments.
 
     This is a wrapper around L{subprocess.check_output}, so it takes
     the same arguments as L{subprocess.Popen} with one difference: all
     arguments after the vector must be keyword arguments.
+
+    @param args: arguments passed to L{subprocess.check_output}
+    @param kwargs: keyword arguments passed to L{subprocess.check_output}
+    @return: command output
+    @rtype: L{str}
     """
     kwargs['stderr'] = STDOUT
-    return check_output(args, **kwargs)
+    output = check_output(args, **kwargs)
+
+    # The check_output() function always returns bytes
+    # on Python 2 and 3, but we want this function to always return
+    # str.
+    if not isinstance(output, str):
+        output = output.decode("utf-8")
+    return output
+
+
+
+class FilePathStrContent(FilePath):
+    """
+    Wrapper around L{FilePath} to make dealing with
+    L{bytes} and L{str}
```

Comment:
I disagree.  The intent of this helper class is to put the bytes logic in one place
just for this module when dealing with FilePath.  If you don't like that, then
the alternative is to go and change all the instances where setContent() / getContent()
is used in this module and do the conversions in those places.

I found it easier to use a helper class.